pkg/cmd/pr/create/create.go

```diff
@@ -229,7 +233,7 @@ func createRun(opts *CreateOptions) (err error) {
 	var openURL string
 
 	if opts.WebMode {
-		if !opts.Autofill {
+		if !(opts.Autofill || opts.FillFirst) {
```

Comment:
This is the only functional change in this PR and appears to have been missed on the initial PR to add the `fill-first` feature. This was the only real change I felt needed to be made but the other clean up changes since I was in the code already.